Tests/EntryTest.swift

```diff
@@ -14,13 +14,13 @@ class EntryTest: XCTestCase {
         XCTAssertEqual(endPoint.pathComponent, "entries")
     }
     #if !NO_FATAL_TEST
-    func testFetch_withoutUID() async {
-        expectFatalError(expectedMessage: "Please provide Entry uid") {
-            makeEntrySut(contentTypeuid: "content").fetch { (result: Result<AssetModel, Error>, response) in
-                        
-            }
-        }
-    }
+//    func testFetch_withoutUID() async {
```

Comment:
@vkalta Could you please clarify the reason for commenting out the test cases block?

Reply:
This is outdated code @sunil-lakshman . This needs to run in the main thread else an error is thrown. So will revisit it later.